conf/modules.config

```diff
@@ -3,39 +3,64 @@
     Config file for defining DSL2 per module options and publishing paths
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Available keys to override module options:
-        ext.args            = Additional arguments appended to command in module.
-        ext.args2           = Second set of arguments appended to command in module (multi-tool modules).
-        ext.args3           = Third set of arguments appended to command in module (multi-tool modules).
-        ext.prefix          = File name prefix for output files.
+        ext.args   = Additional arguments appended to command in module.
+        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix = File name prefix for output files.
 ----------------------------------------------------------------------------------------
 */
 
-process {
+//
+// General configuration options
+//
 
+process {
     publishDir = [
         path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
         mode: params.publish_dir_mode,
         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
     ]
 
-    withName: SAMPLESHEET_CHECK {
+    withName: 'SAMPLESHEET_CHECK' {
         publishDir = [
             path: { "${params.outdir}/pipeline_info" },
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
 
-    withName: FASTQC {
-        ext.args = '--quiet'
-    }
-
-    withName: CUSTOM_DUMPSOFTWAREVERSIONS {
+    withName: 'CUSTOM_DUMPSOFTWAREVERSIONS' {
         publishDir = [
             path: { "${params.outdir}/pipeline_info" },
             mode: params.publish_dir_mode,
             pattern: '*_versions.yml'
         ]
     }
+}
+
+//
+// DBs and parameters preparation options
+//
+
+process {
+    withName: 'UNTAR' {
+        ext.args2 = '--no-same-owner'
+        publishDir = [
+            path: {"${params.outdir}/DBs/${params.mode}/${params.alphafold2_mode}"},
+            mode: 'symlink',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+        ]
+    }
+
+    withName: 'ARIA2' {
+        publishDir = [
+            enabled: false
+        ]
+    }
+}
 
+if (params.mode == 'alphafold2') {
+    includeConfig 'modules_alphafold2.config'
+} else if (params.mode == 'colabfold') {
+    includeConfig 'modules_colabfold.config'
```

Comment:
I think it's better to have that in the main nextflow.config file

Reply:
Why? This will be a configuration that otherwise will be inside the `modules.config` so I'd say that it is better to keep it here as for example in viralrecon see [here](https://github.com/nf-core/viralrecon/blob/3ee1fe98fdf17a80922aa8cf4da4afaf483f3429/conf/modules.config#L41-L45)

Reply:
I'd rather keep the inclusion of files from the main one, instead of including files that include files

Reply:
I see your point, but to me, it makes more sense to include it in modules since hierarchically it belongs to it and also, given that there is already another example in an nf-core pipeline I'd keep the same structure for consistency.